instagram/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import dummyData from './dummy-data';
+import PostContainer from './components/PostContainer/PostContainer.js'
+import SearchBar from './components/SearchBar/SearchBar.js'
+
+class App extends Component {
+	constructor() {
+		super()
+		this.state = {
+			data: dummyData
+		}
+	};
+	
+	
+	
+  render() {
+    return (
+      <div className="App">
+		<h1>INSTAGRAM APP BOIIIII</h1>
+        <SearchBar />
+		{this.state.data.map(data =>(
+			<div key={Math.random()}>
```

Comment:
How would you be able to create a key for `PostContainer` instead of wrapping it around?? **refactoring purposes

Reply:
I need to refactor the entire thing  I believe, i need to change all most functional components to classes. Then I could map inside of postcontainer and use something like key={index}?